src/clj/rems/db/entitlements.clj

```diff
@@ -128,7 +145,10 @@
                            :resource resource-id
                            :revokedby actor
                            :end end})
-    (add-to-outbox! :remove (db/get-entitlements {:application application-id :user user-id :resource resource-id}))))
+    (let [entitlements (db/get-entitlements {:application application-id :user user-id :resource resource-id})]
+      (add-to-outbox! :remove :basic entitlements nil)
```

Comment:
could maybe add a TODO about deprecating the :basic functionality

Reply:
:+1: It's in my notes that I'll probably put in the issue or actually for this create a new one.